index.bs

```diff
@@ -320,18 +315,20 @@ The <dfn method for=IdleDetector>start(|options|)</dfn> method steps are:
     <wpt>
       interceptor.https.html
     </wpt>
-1.  [=Queue a global task=] on the [=relevant global object=] of [=this=] using
-    the [=idle detection task source=] to perform the following steps, but
-    [=abort when=] |this|.{{IdleDetector/[[state]]}} becomes `"stopped"`.
-    1.  Let |permissionState| be the [=permission state=] the [=powerful
-        feature=] named `"idle-detection"`.
-    1.  If |permissionState| is `"denied"`, [=reject=] |result| with a
-        "{{NotAllowedError}}" {{DOMException}}, set
-        |this|.{{IdleDetector/[[state]]}} to `"stopped"` and abort these steps.
+1. [=In parallel=]:
+    1.  Let |permissionState| be the [=permission state=] of `"idle-detection"`.
         <wpt>idle-permission.tentative.https.window.js</wpt>
-    1.  Set |this|.{{IdleDetector/[[state]]}} to `"started"`.
-    1.  Set |this|.{{IdleDetector/[[threshold]]}} to |options|["{{threshold}}"].
-    1.  [=Resolve=] |result|.
+    1.  If |permissionState| is `"denied"`, [=queue a task=] on the
+        [=idle detection task source=] to perform the following steps:
+          1.  Set |this|.{{IdleDetector/[[state]]}} to `"stopped"`.
+          1.  [=Reject=] |result| with a "{{NotAllowedError}}" {{DOMException}}.
+          1.  Abort these steps.
```

Comment:
This abort is in the context of the steps being performed in the new task, not the steps running in parallel. Given the 'If" and "Otherwise" on these two branches I don't think we need an explicit abort here.